index.js

```diff
@@ -0,0 +1,24 @@
+import sass from "sass";
+import { promisify } from "util";
+import { writeFile } from "fs";
+const sassRenderPromise = promisify(sass.render);
+const writeFilePromise = promisify(writeFile);
```

Comment:
I believe you have imported the proper promise-based `writeFile` already. I don't think you have to promisify it.